doxia-modules/doxia-module-markdown/src/main/java/org/apache/maven/doxia/module/markdown/MarkdownSink.java

```diff
@@ -859,6 +873,9 @@ public void nonBreakingSpace() {
 
     @Override
     public void text(String text, SinkEventAttributes attributes) {
+        if (text.equals("\n")) {
```

Comment:
Is this portable POSIX/Windows?

Reply:
Good catch, this is a leftover from some debugging, is not supposed to be in here.

Reply:
It turned out that the HTML parser often emits `text("\n")`without any semantical meaning. It becomes semantical if prefixing that with spaces, though. Therefore added a workaround to not prefix empty line breaks.